webrtc.html

```diff
@@ -1549,81 +1549,104 @@ <h4>Set the RTCSessionDescription</h4>
-                          an existing <a>media transport</a> according to
-                          [[!BUNDLE]], let <var>transport</var> and
-                          <var>rtcpTransport</var> be the
-                          <code><a>RTCDtlsTransport</a></code> objects
-                          representing the RTP and RTCP components of that
-                          transport, respectively.
-                        </p>
-                      </li>
-                      <li>
-                        <p>
-                          Otherwise, let <var>transport</var> and
-                          <var>rtcpTransport</var> be newly created
-                          <code><a>RTCDtlsTransport</a></code> objects, each
-                          with a new underlying
-                          <code><a>RTCIceTransport</a></code>. Though if RTCP
-                          multiplexing is negotiated according to [[!RFC5761]],
-                          or if <var>connection</var>'s
-                          <code><a>RTCRtcpMuxPolicy</a></code> is <code><a
-                          data-link-for="RTCRtcpMuxPolicy">require</a></code>,
-                          do not create any RTCP-specific transport objects,
-                          and instead let <var>rtcpTransport</var> equal
-                          <var>transport</var>.
-                        </p>
-                      </li>
-                      <li>
-                        <p>
-                          Set <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\SenderTransport]]</a>
-                          to <var>transport</var>.
-                        </p>
-                      </li>
-                      <li>
-                        <p>
-                          Set <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\SenderRtcpTransport]]</a>
-                          to <var>rtcpTransport</var>.
-                        </p>
-                      </li>
-                      <li>
-                        <p>
-                          Set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiverTransport]]</a>
-                          to <var>transport</var>.
-                        </p>
-                      </li>
-                      <li>
-                        <p>
-                          Set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiverRtcpTransport]]</a>
-                          to <var>rtcpTransport</var>.
-                        </p>
+                        <p>If <var>description</var> is of type
+                        <code>"answer"</code> or <code>"pranswer"</code>, then
+                        set <var>transceiver</var>'s <a>[[\CurrentDirection]]</a> slot
+                        to an <code><a>RTCRtpTransceiverDirection</a></code>
+                        value representing the direction of the corresponding
+                        <a>media description</a>.</p>
```

Comment:
What about [[PendingDirection]]?

Reply:
I think it ([[PendingDirection]]) should be set to null. I did not do that because it is not used here until it's set again, but forgot its role in 'unmute'.